advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx

```diff
@@ -0,0 +1,185 @@
+ Schema |       Name       | Type  |    Owner
+--------+------------------+-------+--------------
+ public | tp_department_db | table | enterprisedb
+ public | tp_sales_db      | table | enterprisedb
+(2 rows)
+
+
+epas13_test=# select * from  tp_department_db;
+ deptno |    dname    | location
+--------+-------------+----------
+     10 | Development | Pakistan
+     20 | Testing     | Pakistan
+     30 | CM          | Pakistan
+     40 | Marketing   | India
+(4 rows)
+
+
+epas13_test=# select * from  tp_sales_db;
+ salesman_id | salesman_name | sales_region | sales_amount | deptno
+-------------+---------------+--------------+--------------+--------
+         100 | Person 1      | CITY 1       |            1 |     10
+         110 | Person 2      | CITY 2       |            2 |     20
+         120 | Person 3      | CITY 3       |            3 |     30
+         130 | Person 4      | CITY 4       |        10000 |     40
+(4 rows)
+
+
+epas13_test=# select * from v1;
+ dept_no |  dept_name  | sales_no | sales_name | sales_salary | sales_dept_no
+---------+-------------+----------+------------+--------------+---------------
+      10 | Development |      100 | Person 1   |            1 |            10
+      20 | Testing     |      110 | Person 2   |            2 |            20
+      30 | CM          |      120 | Person 3   |            3 |            30
+      40 | Marketing   |      130 | Person 4   |        10000 |            40
+(4 rows)
+
+
+epas13_test=# desc  tp_sales_db;
+                                 Table "public.tp_sales_db"
+    Column     |         Type          | Collation | Nullable |           Default
+---------------+-----------------------+-----------+----------+------------------------------
+ salesman_id   | integer               |           |          |
+ salesman_name | character varying(30) |           |          |
+ sales_region  | character varying(30) |           |          |
+ sales_amount  | integer               |           |          | nextval('sal_seq'::regclass)
+ deptno        | integer               |           |          |
+Indexes:
+    "lower_reg_idx" btree (lower(sales_region::text))
+    "reg1_idx" btree (salesman_id)
+Foreign-key constraints:
+    "department_employee_fk" FOREIGN KEY (deptno) REFERENCES tp_department_db(deptno)
+
+
+epas13_test=#
+epas13_test=#
+
+```
+
+### FSBasedBackupSet Backup and Restore
+
+At this time there is a known issue with FSBased Restore that does not allow for proper restoration of the database. See the known issues section of the guide for more information.
+
+#### Known Issues:
+1. FSBaseBadkupSet Restore has an issue if the *:\Program files\edb has been deleted, after the restore is done, the permissions on the restored directories are not recovered, and it instead inherits the permissions from the parent directory which does not allow EDB Postgres Advanced Server services to start on the restored directory. We are working with Commvault to resolve the issue.
```

Comment:
replace with
FSBaseBadkupSet Restore has an issue if the default edb directory for example *:\Program files\edb has been lost or deleted.  If this occurs then after a restore is performed, the permissions on the restored directories are not recovered. Instead the directory inherits the permissions from the parent directory which does not allow EDB Postgres Advanced Server services to start on the restored directory. We are working with Commvault to resolve the issue.